templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
@wfernandes if you intend to use `client_cert` and `client_key` namespaced to your client jobs, why does the PR add these properties to the top-level `etcd` namespace?